app/views/explore/index.html.erb

```diff
@@ -0,0 +1,160 @@
+            data-action="click->explore#switch"
+            data-key="picks"
+            role="tab" aria-selected="false"><%= t("explore.tabs.picks") %></button>
+    <button class="explore-tab"
+            data-explore-target="tab"
+            data-action="click->explore#switch"
+            data-key="recent"
+            role="tab" aria-selected="false"><%= t("explore.tabs.recent") %></button>
+    <button class="explore-tab"
+            data-explore-target="tab"
+            data-action="click->explore#switch"
+            data-key="tags"
+            role="tab" aria-selected="false"><%= t("explore.tabs.tags") %></button>
+  </div>
+
+  <section class="explore-section" data-explore-target="section" data-key="cotw">
+    <div class="row center-row home-circuit-card-row">
+      <div class="col-12">
+        <h3><%= t("explore.sections.cotw.heading") %></h3>
+        <p><%= t("explore.sections.cotw.description") %></p>
+      </div>
+    </div>
+    <% if @circuit_of_the_week %>
+      <div class="project-cards-grid">
+        <%= render(Project::ProjectCardComponent.new(project: @circuit_of_the_week)) %>
+      </div>
+    <% else %>
+      <p class="text-muted"><%= t("explore.empty") %></p>
+    <% end %>
+  </section>
+
+  <section class="explore-section d-none" data-explore-target="section" data-key="examples">
+    <div class="row center-row">
+      <div class="col-12">
+        <h3><%= t("explore.sections.examples.heading") %></h3>
+        <p><%= t("explore.sections.examples.description") %></p>
+      </div>
+    </div>
+    <div class="row center-row examples-container">
+      <% @examples.each do |example| %>
+        <div class="col-12 col-sm-12 col-md-6 col-lg-4">
+          <div class="card circuit-card">
+            <%= image_tag example[:img], alt: example[:name], class: "card-img-top circuit-card-image examples-circuit-card-image" %>
+            <div class="card-footer circuit-card-footer">
+              <div class="circuit-card-name">
+                <p><%= example[:name] %></p>
+                <span class="tooltiptext"><%= example[:name] %></span>
+              </div>
+              <a class="btn primary-button circuit-card-primary-button"
+                 target="_blank" rel="noopener noreferrer"
+                 href="https://circuitverse.org/<%= example[:id] %>"><%= t("view") %></a>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </section>
+
+  <section class="explore-section d-none" data-explore-target="section" data-key="picks">
+    <div class="row center-row">
+      <div class="col-12">
+        <h3><%= t("explore.sections.picks.heading") %></h3>
+        <p><%= t("explore.sections.picks.description") %></p>
+      </div>
+    </div>
+    <% if @editor_picks.any? %>
+      <div class="project-cards-grid">
+        <% @editor_picks.each do |project| %>
+          <%= render(Project::ProjectCardComponent.new(project: project)) %>
+        <% end %>
```

Comment:
Use `with_collection` to render array of components

https://viewcomponent.org/guide/collections.html